src/app/clusters/localization-time-format-server/localization-time-format-server.cpp

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
The cluster is called "Time Format Localization" in the spec.  Why are we calling it "Localization Time Format" everywhere (including in the naming of files, the C++ comments, and the cluster name in the XML) instead of the spec name?

Reply:
No idea, it is not introduced by this PR, I will create a PR to align all names to the spec

Reply:
I think the spec is not consistent on this cluster. It is defined as LocalizationTimeFormat in the 1.1.3. Cluster Identifiers

1.1.3 Identifier | Name
-- | --
0x002c | LocalizationTimeFormat